### DISCUSSION TITLE
Subdomains

### DISCUSSION THREAD

**Discussion by GomorrhaDev** [Paymenter/Paymenter#222] [Ideas]
Hi,
It would be quite cool if the buyer can create a subdomain when buying through Paymenter.
This can be achieved with the Cloudflare API (you have to have changed the nameservers with your own domain to Cloudflare's of course) and SRV records, which the buyer can then use to connect to the game server, which is on a specific port.

Personally, this would save me a lot of time, as I currently have to manually edit my DNS record at the request of users.

(I would pay 10€ for this function, thats 1€ per working hour i guess (: )

**Comment by BasToTheMax** [comment]
Not everyone uses cloudflare tho

**Reply by GomorrhaDev** [reply]
I know but Cloudflare is the only platform I know where you can change the DNS records via an API. Also, if you want to have a domain for it, you can easily move it to Cloudflare.

**Reply by cloudrack-ca** [reply]
ah ptero negated comment

**Comment by Xteri08** [comment]
If you are using games to host its gona be a panel and there are already such addons you can use even in ptero too i use a subdomain manager which works with cloudflare

**Reply by GomorrhaDev** [reply]
Yes, but it would also be quite cool if you could integrate something like a payment function if you want to use a subdomain.
